dockers/docker-database/database_global.json.j2

```diff
@@ -26,7 +26,7 @@
 {% if dpu_cnt > 0 %}
 {% for dpu in range(dpu_cnt) %}
         {
-            "database_type" : "dpudb",
+            "database_name" : "dpu{{dpu}}",
```

Comment:
Sorry for late comment. "database" will be ambigous terminology since there are multiple databases inside one Redis instance. How about "container_name"